grind/python/disttest/isolate.py

```diff
@@ -60,11 +60,12 @@ class Isolate:
 
     __ISOLATE_NAME = """disttest.isolate"""
 
-    def __init__(self, project_root, output_dir,
+    def __init__(self, project_root, output_dir, java_version=None,
                  include_modules=None, exclude_modules=None, include_patterns=None, exclude_patterns=None,
                  cache_dir=None, extra_deps=None, maven_flags=None, maven_repo=None, verbose=False):
         logger.info("Using output directory " + output_dir)
         self.output_dir = output_dir
+        self.java_version = java_version
```

Comment:
I'd like some kind of assert that this is not null for safety, since otherwise we won't generate a valid run script.